### ISSUE TITLE
Make features and annotations into experiment data

### ISSUE THREAD

**Issue by jixu7952** [ciaua/clip2frame#1]
missing script?

**Comment by ciaua** [comment]
The features of all 6 scales from MagnaTagATune will take quite a lot of spaces. I will see what I can do.

**Comment by jixu7952** [comment]
a few sample lines of data to show the npy file format would be desirable, and the python file for making experiment data is missing

**Comment by TobyOoO** [comment]
Yes, I am confused about how to retrieve tags backward from outcomes of the model.

**Comment by ciaua** [comment]
Thank you both for your comments. I think I will take this chance to overhaul the whole package, including updating the codes, uploading the experiment data. 

I will also make the feature standardization process independent of the scikit-learn package by including the standardization directly into the model. The standardizer module in scikit-learn has been changing several times recently, so doing so will make the maintenance easier in case it changes again.

**Comment by jixu7952** [comment]
thanks for update, Ciaua. Could you please kindly share the script to prepare the experiment data?

**Comment by ciaua** [comment]
I have added `process_annotations.top_n_tags.py` and `make_exp_data.py`.
You still have to download the MagnaTagATune files from their website though.
